### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by rbignon**

### CODE REVIEW

**Comment by ltworf** [comment]
"fix tests" can't be a pull request on its own… it means I first merged a thing that broke them…

File: localslackirc/irc.py [hunk 1]
```diff
@@ -991,13 +990,9 @@ async def irc_listener() -> None:
                 to_irc_task,
             )
         finally:
-            log('Closing connections')
-            sl_client.close()
-            if status_file:
-                log(f'Writing status to {status_file}')
-                status_file.write_bytes(sl_client.get_status())
```
Comment (ltworf) [inline localslackirc/irc.py]: Case in point: this commit claims to touch logging, yet it removes saving the status file.

Big surprise that I didn't accept your changes…

Comment (rbignon) [reply localslackirc/irc.py]: I suggest you to have a look at the end of daemon.py.

Comment (rbignon) [reply localslackirc/irc.py]: But you're right there was a bug, shutdown wasn't correctly handled, and the status file wasn't written.

Fix: https://github.com/rbignon/localslackirc/commit/8bb410ab6c819708e975e7fd16ebdc563828ba6c

**Pull request closed without merging**